actionSHARK/utils.py

```diff
@@ -3,29 +3,25 @@
 from pycoshark.mongomodels import Commit, JobStep, RunPullRequest
 import sys
 import datetime as dt
+import dateutil
 
 
-def parse_date(value: Optional[str] = None, is_millisecond: bool = False
-) -> Optional[dt.datetime]:
+def parse_date(value: Optional[str] = None) -> Optional[dt.datetime]:
     """Convert Datetime string to actual Datetime object.
 
     Args:
         value (str): The datetime string to be converted. Defaults to None.
-        is_millisecond (bool, optional): If datetime has millisecond in the string. Defaults to False.
 
     Returns:
         datetime: datetime formated or None if no string passed.
     """
     if not value:
         return None
 
-    date_fmt = "%Y-%m-%dT%H:%M:%S%z"
-    if is_millisecond:
-        date_fmt = "%Y-%m-%dT%H:%M:%S.%f%z"
+    date = dateutil.parser.parse(value)
+    date = date.replace(tzinfo=None)
```

Comment:
Did something change that this is needed? What is the result of the replace operation? Does it convert the date from some timezone to utc or just removes the timezone?

Reply:
I tested it. It does not change the time it will just remove the info. Either way, timezone info will not stored in the Database.

Reply:
If that is the case please add a date.astimezone(pytz.UTC) before the replace. If we already get UTC from the API nothing will change and if we get a different timezone it is converted to UTC before we remove the timezone information.

Reply:
```suggestion
     import pytz
    date = date.astimezone(pytz.UTC).replace(tzinfo=None)
```